src/collectd.conf.pod

```diff
@@ -10596,6 +10596,141 @@ C<https://monitoring.googleapis.com/v3>.
 
 =back
 
+=head2 Plugin C<write_syslog>
+
+The C<write_syslog> plugin writes data in I<syslog> format log messages.
+It implements the basic syslog protocol, extends it with
+content-based filtering, rich filtering capabilities,
+flexible configuration options and adds features such as using TCP for transport.
+The plugin can connect to a I<Syslog> daemon that will ingest metrics, transform and
```

Comment:
Perhaps give an example of such a daemon. Which daemons support this input? rsyslog, syslog-ng?

Reply:
I can provide a simple syslog-ng config

Reply:
Please...